libs/facilities/backend/timeseries/src/lib/interfaces/request.interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface IGetTimeSeriesParams {
+	entityId: string;
+	propertySetName: string;
+}
```

Comment:
I would prefer 2 distinct files and a short docs, however I understand that it could be over engineering